src/IonParserTextRaw.ts

```diff
@@ -263,6 +269,18 @@ export class ParserTextRaw {
     helpers[CH_MS] = this._read_value_helper_minus; // '-'
 
     this._read_value_helper_helpers = helpers;
+
+    helpers = {};
```

Comment:
I would prefer not to reuse the var by re-initializing to `{}`. 
Is it true that for the case of tokens there is really one extra helper compared to the case for values? 

 I would refactor out a common object and add the variance for each set of helpers.